App.py

```diff
@@ -2,7 +2,7 @@
 import logging  # Error handling and recording adding/editing/removing a book.
 from flask import Flask, render_template, request, redirect, url_for
 from Personal_Library_Manager import MyLibrary, Book
-
+#
 # Configure logging
 logging.basicConfig(level=logging.DEBUG)
 app = Flask(__name__)

```

Comment:
You need to add a message when trying to enter invalid values, in add book i tried to add invalid queires nothing happened i didn't understand what happened